<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_recipient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter recipient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addRecipient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add recipient</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_recipients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_body">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Body</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendEmail">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND EMAIL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Logout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
